content/en/docs/Installation/local-installation.md

```diff
@@ -0,0 +1,99 @@
+<!--
+---
+title: "Local kubernetes cluster"
+linkTitle: "Local cluster"
+weight: 1
+description: >
+  Set up a Kubernetes cluster on your computer to test Tekton.
+---
+-->
+
+If you are going to test Tekton locally on your computer, two possible
+alternatives for that are **minikube** and **kind**. 
```

Comment:
There are other alternatives as well, like k3c, but of course we do not need to document all.
The problem I see here is that these are vanilla instructions, there's no Tekton specific setup / tuning, so I don't see the value in maintaining them here as opposed to just link to `kind` and `minikube` docs.

If we were to say how setup a cluster for Tekton development, or how to setup a cluster with a local registry for pushing images in Tekton tasks, I think there'd be more value, but as it is I don't see this content providing help.